DEVELOPMENT.md

```diff
@@ -0,0 +1,46 @@
+### Debugging test
+
+#### MacOS
+
+You could debug test with [Delve](https://github.com/go-delve/delve) debugger.
+Run:
+```shell
+make test-mac-debug
+```
+This command will run build `gvisor` binary with debugger enabled.
+
+>Note: By default it would use `--continue` `dlv` option to not pause `givisor` execution on start, if debugger is not connected.
```

Comment:
`givisor` typo (extra `i`). What do you mean by gvisor though? Is it gvproxy, or is it any program using gvisor-tap-vsock as a package?

Reply:
It is binary name, which we use during test